### ISSUE TITLE
unexpected token: TRIGGERS

### ISSUE THREAD

**Issue by movedoa** [Riverside-Software/sonar-openedge#284]
Version 2.0.2
Very easy to reproduce with the following code.

```
DEFINE VARIABLE lcFoo AS CHARACTER NO-UNDO.
DEFINE VARIABLE lhFoo1 AS HANDLE NO-UNDO.

CREATE VALUE(lcFoo) lhFoo1 TRIGGERS:

END.
```

**Comment by movedoa** [comment]
I found a nearly identical error with assign. (unexpected token: ASSIGN)
```
   DEFINE VARIABLE lcFoo AS CHARACTER NO-UNDO.
   DEFINE VARIABLE lhFoo1 AS HANDLE NO-UNDO.
   DEFINE VARIABLE lhFrame AS HANDLE NO-UNDO.

   CREATE VALUE(lcFoo) lhFoo1
     ASSIGN
       FRAME = lhFrame.
```

**Comment by gquerret** [comment]
Probably linked to #268

**Comment by gquerret** [comment]
Fixed in master

**Comment by gquerret** [comment]
Issue came from a regression after having fixed #268
This regression could have been caught if only warnings in the ANTLR phase were caught correctly 🙄  Moral of the story: always have a look at warnings
